tests/test_git_abort.py

```diff
@@ -0,0 +1,72 @@
+import os
+import subprocess
+import shlex
+import git
+from helper import invoke_git_extras_command
+
+class TestGitAbort:
+    def test_init(self, git_repo):
+        repo, tmp_dir, tmp_a, tmp_b = git_repo
+        repo.git.branch("A")
+        repo.git.branch("B")
+        repo.git.checkout("A")
+        with open(tmp_a[1], "w", encoding="utf-8") as file_a_on_A:
+            file_a_on_A.write("a")
+            file_a_on_A.close()
+        repo.git.add(".")
+        repo.git.commit("-m", "A")
+        repo.git.checkout("B")
+        with open(tmp_a[1], "w", encoding="utf-8") as file_a_on_B:
+            file_a_on_B.write("b")
+            file_a_on_B.close()
+        repo.git.add(".")
+        repo.git.commit("-m", "B")
+        repo.git.status()
+
+    def test_cherry_pick(self, git_repo):
+        repo = git_repo[0]
+        try:
+            repo.git.cherry_pick("A")
+        except git.GitCommandError:
+            pass
```

Comment:
Let's print the error if the cmd failed unexpectedly

Reply:
Dump the error when unexpected.